README.md

```diff
@@ -20,7 +20,7 @@ yarn add @internetarchive/ia-menu-slider
   import 'ia-book-search-results/ia-book-search-results.js';
 </script>
 
-<ia-book-search-results></ia-book-search-results>
+<ia-book-search-results .bookReader=${window.br}></ia-book-search-results>
```

Comment:
You've got the book reader still being passed in here, in `custom-elements.json`, and in the tests.